docs/src/development/cli/_index.md

```diff
@@ -1,148 +1,161 @@
 ---
-title: "CLI (Command line interface)"
+title: "Use the command line interface (CLI)"
 weight: 3
 description: |
-  The CLI is the official tool to use and manage your Platform.sh projects directly from your terminal. Anything you can do within the management console can be done with the CLI.
-sidebarTitle: "CLI"
+  See how to use and manage your Platform.sh projects directly from your terminal. Anything you can do within the management console can be done with the CLI.
+sidebarTitle: "Use the CLI"
 layout: single
 ---
 
 {{< description >}}
 
-Behinds the scenes it uses both the Git interface and our REST API. The source code of the CLI is hosted on [GitHub](https://github.com/platformsh/platformsh-cli).
+The CLI uses the git interface and the [Platform.sh REST API](https://api.platform.sh/docs/) to accomplish tasks.
+Its source code is hosted on [GitHub](https://github.com/platformsh/platformsh-cli).
 
-Find detailed information on [setting up a local development environment](/gettingstarted/developing/local-development/_index.md).
+## Before you begin
 
-## Installation
+You need to have:
+
+* PHP 5.5.9+
+* Git
+* A Bash-like shell
+
+  On Windows, the best way to get Bash is through [Windows Subsystem for Linux](https://msdn.microsoft.com/en-gb/commandline/wsl/about).
+  You can also use another Bash-compatible shell such as [Git Bash](https://gitforwindows.org/),
+  which you might have installed with git.
+
+For full requirements, see the [requirements on GitHub](https://github.com/platformsh/platformsh-cli#requirements).
+
+## 1. Install
 
 Install the CLI using this command:
 
 ```bash
 curl -fsS https://platform.sh/cli/installer | php
 ```
 
-You can find the system requirements and more information in the [installation instructions on GitHub](https://github.com/platformsh/platformsh-cli/blob/master/README.md#installation).
+If you run into issues or want to install manually, see the [installation instructions on GitHub](https://github.com/platformsh/platformsh-cli#installation).
 
-## Authentication
+## 2. Authenticate
 
-The [Platform.sh CLI](https://github.com/platformsh/platformsh-cli) will authenticate you with Platform.sh and show your projects. Run this command to start:
+To see and manage projects, you need to be authenticated with Platform.sh
+Run this command:
 
 ```bash
 platform
 ```
 
-You will be asked to log in via a browser.
-
-When you are logged in, a list of your projects appears, along with some tips for getting started.
-
-**Your command-line tools are now ready to use with Platform.sh.**
+You are asked to log in via a browser.
+This process creates SSH certificates on your computer for secure connections.
+(See more about [SSH connections](/development/ssh/_index.md).)
```

Comment:
Id leave it without the ( ). See more about [SSH connections](/development/ssh/_index.md)

Reply:
This information is rather outside the main flow of the article. I don't think it's important enough to warrant being put in a Note or anything, but I would indicate it's less important. I think the parentheses do a good job of that.